src/ObjCRuntime/Dlfcn.cs

```diff
@@ -211,6 +211,7 @@ public static ulong GetUInt64 (IntPtr handle, string symbol)
 			return (ulong) Marshal.ReadInt64 (indirect);
 		}
 
+#if !XAMCORE_4_0
```

Comment:
in general we put `[Obsolete]` on API we plan to remove
not sure it matter much in this case, the compiler should pick up the appropriate one...

Reply:
@spouliot I've added the [Obsolete].